composer.json

```diff
@@ -35,7 +34,7 @@
   },
   "autoload-dev": {
     "psr-4": {
-      "TreeHouse\\QueueBundle\\Tests\\": "tests/src/TreeHouse/QueueBundle/"
```

Comment:
Can we keep the `Tests` suffix to prevent duplicate namespace definitions in the autoloader? It will still work this way but I prefer it if the tests only can resolve to a specific directory.

Reply:
But that's not PSR-4 compliant. Is that ok?

Reply:
This should be, right?

```
"TreeHouse\\QueueBundle\\Tests\\": "tests/src/TreeHouse/QueueBundle/Tests/"
```

Reply:
yup!